### PR TITLE
Datahub 9.0 url update

### PR DESCRIPTION
**Pull request by jakubneubauer**

### CODE REVIEW

**Comment by LosD** [comment]
LGTM, though I have no idea if the paths are correct :)

**Comment by LosD** [comment]
If you're sur about the paths,, merge when Travis is happy.

File: engine/core/src/main/java/org/datacleaner/metamodel/datahub/DataHubUpdateConnection.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
  * Note: Some REST controllers do not need the tenant info. Others do.
  */
 public class DataHubUpdateConnection {
-    public final static String CONTEXT_PATH = "/service/v1";
+    public final static String CONTEXT_PATH = "/api/v1";
```
Comment (reinierdejong) [inline engine/core/src/main/java/org/datacleaner/metamodel/datahub/DataHubUpdateConnection.java]: should be /datahub/api/v1

**Comment by jakubneubauer** [comment]
Now it is correct.

**Comment by LosD** [comment]
Cool. Merging!

**Pull request merged**